design-system/react/src/components/Accordion/AccordionItem.tsx

```diff
@@ -1,23 +1,19 @@
-import { cx } from '@fuel-ui/css';
 import * as AC from '@radix-ui/react-accordion';
+import { createElement } from 'react';
 
-import { createComponent, createStyledElement } from '../../utils';
+import { createComponent2 } from '../../utils';
```

Comment:
Can we rename the old method to `createComponentOld`?

Reply:
I just rename the new one to `_unstable_createComponent`